### ISSUE TITLE
[Bug] Stocks/Search returns strange results and lots of empty fields.

### ISSUE THREAD

**Issue by deeleeramone** [OpenBB-finance/OpenBBTerminal#4080]
I get a whole bunch of results returned with None for every field. 

```
2023 Jan 30, 12:36 (🦋) /stocks/ $ search --exchange united_kingdom

             Companies found on an exchange in United Kingdom              
┏━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━┓
┃         ┃ Name             ┃ Country     ┃ Sector ┃ Industry ┃ Exchange ┃
┡━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━┩
│ 0A00.IL │ None             │ None        │ None   │ None     │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A02.IL │ None             │ None        │ None   │ None     │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A05.L  │ Medacta Group SA │ Switzerland │        │          │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A0C.L  │ Stadler Rail AG  │ Switzerland │        │          │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A0D.IL │ None             │ None        │ None   │ None     │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A0E.IL │ None             │ None        │ None   │ None     │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A0F.IL │ None             │ None        │ None   │ None     │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A0H.IL │ None             │ None        │ None   │ None     │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A0I.IL │ None             │ None        │ None   │ None     │ None     │
├─────────┼──────────────────┼─────────────┼────────┼──────────┼──────────┤
│ 0A0J.IL │ None             │ None        │ None   │ None     │ None     │
└─────────┴──────────────────┴─────────────┴────────┴──────────┴──────────┘
```

Loading one of these 'None' tickers, 0A0D.IL,

```
2023 Jan 30, 12:42 (🦋) /stocks/fa/ $ load 0A0D.IL --exchange

Loading Daily data for 0A0D.IL with starting period 2021-02-10.

Company:  ALCON AG ALCON ORD SHS
Exchange: IOB
Currency: CHF
```

`/fa/info` shows the same data as: https://finance.yahoo.com/quote/0A0D.IL/profile?p=0A0D.IL

![Screenshot 2023-01-30 at 9 52 57 AM](https://user-images.githubusercontent.com/85772166/215555274-db9217d0-0083-42e2-b960-53b3bc947693.png)

Looking at the actual website for the Exchange provides some data as well:

https://www.londonstockexchange.com/securities-trading/equity-trading/international-order-book 

https://www.londonstockexchange.com/market-stock/0A0D/alcon-ag/trade-recap

**Comment by JerBouma** [comment]
Fixing this issue here: https://github.com/OpenBB-finance/OpenBBTerminal/pull/4084. It's not really a bug, it's simply that those tickers lack data within my database while also not really being too important to look at.

**Comment by JerBouma** [comment]
This has been fixed in #4084 and in v2.3

```
(🦋) /stocks/ $ search --exchange united_kingdom

                                             Companies found on an exchange in United Kingdom
┏━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┓
┃        ┃ Name                                     ┃ Country        ┃ Sector                 ┃ Industry                       ┃ Exchange ┃
┡━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━┩
│ AML.L  │ Aston Martin Lagonda Global Holdings plc │ United Kingdom │ Consumer Cyclical      │ Auto Manufacturers             │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ CYBA.L │ Cyba Plc                                 │ United Kingdom │ Financial Services     │ Shell Companies                │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ D4T4.L │ D4t4 Solutions Plc                       │ United Kingdom │ Technology             │ Software - Application         │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ DAL.L  │ Dalata Hotel Group plc                   │ Ireland        │ Consumer Cyclical      │ Lodging                        │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ DARK.L │ Darktrace plc                            │ United Kingdom │ Technology             │ Software - Infrastructure      │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ DATA.L │ GlobalData Plc                           │ United Kingdom │ Industrials            │ Consulting Services            │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ DBOX.L │ Digitalbox plc                           │ United Kingdom │ Communication Services │ Internet Content & Information │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ DC.L   │ Dixons Carphone plc                      │ United Kingdom │ Consumer Cyclical      │ Specialty Retail               │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ DCC.L  │ DCC plc                                  │ Ireland        │ Energy                 │ Oil & Gas Refining & Marketing │ LSE      │
├────────┼──────────────────────────────────────────┼────────────────┼────────────────────────┼────────────────────────────────┼──────────┤
│ DCD.L  │ DCD Media Plc                            │ United Kingdom │ Communication Services │ Entertainment                  │ LSE      │
└────────┴──────────────────────────────────────────┴────────────────┴────────────────────────┴────────────────────────────────┴──────────┘
```
